server/web/grace/grace.go

```diff
@@ -107,6 +107,10 @@ func init() {
 
```

Comment:
Pls use Options mode rathan making anthor method
like
```golang
func NewServer(addr string, handler http.Handler,opts ... ServerOpt) (srv *Server) {
```

Reply:
`func NewServer(addr string, handler http.Handler,opts ... ServerOpt)` is adopted.